template.yaml

```diff
@@ -0,0 +1,84 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  Template creates lambda, api-gateway and custom domain mapping for fetching person data
+
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: Person
+    Description: Proxy for fetching person data from the Cristin API
+    Author: Unit
+    SpdxLicenseId: MIT
+    LicenseUrl: LICENSE
+    Labels: ['${CODEBUILD_RESOLVED_SOURCE_VERSION}', '${GIT_REPO}', '@${BUILD_TIMESTAMP}']
+
+Globals:
+  Function:
+    Timeout: 20
+  Api:
+    Cors:
+      AllowOrigin: "'*'"
+      AllowMethods: "'OPTIONS, GET, PUT'"
+      AllowHeaders: "'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token, Access-Control-Allow-Origin'"
+
+Parameters:
+  CognitoAuthorizerArn:
+    Type: String
+    Description: Reference to Cognito UserPool for the stage
+  CustomDomain:
+    Type: String
+    Description: Custom API to connect this lambda to
+  CustomDomainBasePath:
+    Type: String
+    Description: Base path mapping in CustomDomain
+
+Resources:
+
+  ApiAccessLogGroup:
+    Type: AWS::Logs::LogGroup
+
+  NvaCristinPersonFunctionApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      AccessLogSetting:
+        DestinationArn: !GetAtt ApiAccessLogGroup.Arn
+        Format: '{ "apiId": "$context.apiId", "requestId": "$context.requestId", "requestTime": "$context.requestTime", "requestTimeEpoch": "$context.requestTimeEpoch", "httpMethod": "$context.httpMethod", "path": "$context.path", "status": "$context.status",  "error.message": "$context.error.message" }'
+      StageName: Prod
+      Auth:
+        Authorizers:
+          MyCognitoAuthorizer:
+            UserPoolArn: !Ref CognitoAuthorizerArn
+      EndpointConfiguration: Regional
+
+
+  NvaCristinPersonFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Environment:
+        Variables:
+          ALLOWED_ORIGIN: '*'
+          CRISTIN_API_HOST: 'api.cristin.no'
+          BASE_URL: 'https://api.dev.nva.aws.unit.no/person'
+      Handler: no.unit.nva.cristin.person.FetchCristinPerson::handleRequest
```

Comment:
Is this correct? Have you tested using sam local start-api? I think it should be CristinProjectsHandler instead of FetchCristinPerson?

Reply:
fixed.